src/CommunityToolkit.Maui/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System.ComponentModel;
```

Comment:
The file name should have suffix "shared" like other files.

Reply:
Just pushed changes addressing this.